src/components/ParentNavigationSubtitle.js

```diff
@@ -18,7 +18,7 @@ const propTypes = {
     }).isRequired,
 
     /** parent Report ID */
-    parentReportID: PropTypes.string,
+    parentReportID: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
Hmm I thought reportID were always supposed to be strings when they got returned from the API... if this isn't the case something might be broken in the API.

Reply:
https://expensify.slack.com/archives/C01GTK53T8Q/p1691167905181059

Reply:
I think we are just missing cases in the backend where these properties should be converted to strings... like `parentReportID`

For example, the `reportID` in a report is a string:

<img width="523" alt="image" src="https://github.com/Expensify/App/assets/87341702/67085a13-6b86-41ea-866c-8f7ebf4eaf44">

We should be consistent on that.


Now... since the backend is wrong, I'm not sure what do in your case... should we fix the backend and put this on hold until that is done? or we fix this in a follow up when we update the backend?

Reply:
@bondydaa Could you please confirm that we should revert this [commit](https://github.com/Expensify/App/pull/24113/commits/d07e5c133608eba929a9c3973b1b8800bd552b85)?

Reply:
Here is a link to the related bug: https://expensify.slack.com/archives/C049HHMV9SM/p1691035111039979.

Reply:
@bondydaa @fedirjh Could you please confirm that we should revert this [commit](https://github.com/Expensify/App/pull/24113/commits/d07e5c133608eba929a9c3973b1b8800bd552b85)?

Reply:
Yeah let's revert that, we have a PR open right now to fix this so that the BE will return strings.

Reply:
@bondydaa I have reverted last commit. Can you please review again?